src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -177,19 +180,19 @@ class PeerActor(
       daoBlockHeaderOpt match {
         case Some(_) if daoForkValidator.validate(msg).isEmpty =>
           log.info("Peer is running the ETC chain")
-          context become new HandshakedHandler(rlpxConnection, remoteStatus).receive
+          context become new HandshakedHandler(rlpxConnection, remoteStatus, daoForkBlockNumber).receive
 
         case Some(_) if nodeStatusHolder().blockchainStatus.totalDifficulty < daoForkBlockTotalDifficulty =>
           log.info("Peer is not running the ETC fork, but we're not there yet. Keeping the connection until then.")
-          context become new HandshakedHandler(rlpxConnection, remoteStatus).receive
+          context become new HandshakedHandler(rlpxConnection, remoteStatus, daoForkBlockNumber).receive
```

Comment:
but it responded with block header that has number `daoForkBlockNumber` so we should send to it only blocks that have number grater then `daoForkBlockNumber`
@LukasGasior1 shouldnt we disconnect from the peer that is not running on ETC chain?